tgui/packages/tgui/interfaces/BrigCell.js

```diff
@@ -0,0 +1,203 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Box, ColorBox, NoticeBox, Flex, ProgressBar, Button, LabeledList, Divider } from '../components';
+
+export const BrigCell = (props, context) => {
+  const { data, act } = useBackend(context);
+  const { viewing_incident, incidents } = data;
+
+  return (
+    <Window theme="weyland" width={450} height={450}>
+      <Window.Content scrollable>
+        <Flex justify="center" mt="2rem" mb=".5rem">
+          <Button
+            content="Toggle Shutters"
+            height="100%"
+            mr="1rem"
+            px="2rem"
+            py=".25rem"
+            onClick={() => act('toggle_doors')}
+          />
+          <Button
+            content="Activate Flash"
+            height="100%"
+            px="2rem"
+            py=".25rem"
+            onClick={() => act('flash')}
+          />
+        </Flex>
+
+        <Divider />
+
+        <Flex direction="column" justify="center" align="center" width="auto">
+          {!viewing_incident && (
+            <>
+              <h1>Sentences:</h1>
+              <Flex align="center">
+                <ColorBox color="green" />
+                <Box inline px="1rem">
+                  Active
+                </Box>
+              </Flex>
+
+              {incidents.map((incident, i) => (
+                <Button
+                  key={i}
+                  textAlign="center"
+                  minWidth="15rem"
+                  p=".5rem"
+                  my=".5rem"
+                  color={incident.active ? 'green' : ''}
+                  onClick={() =>
+                    act('set_viewed_incident', { incident: incident.ref })
+                  }>
+                  {incident.suspect}
+                </Button>
+              ))}
+            </>
+          )}
+        </Flex>
+
+        {!!viewing_incident && <IncidentDetails />}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const IncidentDetails = (props, context) => {
+  const { data, act } = useBackend(context);
+  const {
+    suspect,
+    progress,
+    sentence,
+    time_left,
+    started,
+    active,
+    status_class,
+    status_text,
+    can_pardon,
+  } = data;
+
+  return (
+    <>
+      <Flex mb=".5rem" justify="space-between" align="center">
+        <Button
+          icon="arrow-left"
+          px="2rem"
+          py=".25rem"
+          mr="1rem"
+          onClick={() => act('set_viewed_incident')}
+        />
+
+        <Flex.Item>
+          {status_text !== 'PERMABRIG' && (
```

Comment:
all these PERMABRIG checks could be done like `const Permabrig = status_text === 'PERMABRIG';`
would be a bit easier to read/write